testdata/go/v3/memcached-operator/docs/monitoring/metrics.md

```diff
@@ -0,0 +1,11 @@
+# Memcached Operator Metrics
+This document aims to help users that are not familiar with metrics exposed by the memcached-operator.
+All metrics documented here are auto-generated by the utility tool `monitoring/metricsdocs` and reflects exactly what is being exposed.
+
+## Memcached Operator Metrics List
+### memcached_deployment_size_undesired_count_total
+Total number of times the deployment size was not as desired. Type: Counter.
+## Developing new metrics
+After developing new metrics or changing old ones, please run `make generate-metricsdocs` to regenerate this document.
+
+If you feel that the new metric doesn't follow these rules, please change `tools/doc-generator/monitoring/metrics` with your needs.
```

Comment:
@avlitman I think you should update
with your needs -> according to your needs

Reply:
+1 and also the path is wrong. fixing